docs/administering/OFFBOARDING_A_FUND.md

```diff
@@ -0,0 +1,167 @@
+  post_deletion_counts[fund.name] = fund_counts                        
+end                                                                    
+fund_counts = {}                                                       
+models_fund_agnostic.each do |model|                                   
+  fund_counts[model.to_s.to_sym] = model.all.size                      
+end                                                                    
+post_deletion_counts['fund_agnostic'] = fund_counts                    
+                                                                       
+#save starting counts to file                                          
+$stdout = File.new('post_deletion.out', 'w')                           
+$stdout.sync = true                                                    
+post_deletion_counts                                                   
+$stdout = STDOUT
+
+exit
+```
+
+diff those files!
+
+  - The other fund objects should not have changed significantly
+  - There should be a jump in PaperTrails corresponding to the total object count we've deleted
+
+`diff -y initial_count.out post_deletion.out`
+
+IF AND ONLY IF everything looks good, let's continue.
+Remove the PaperTrails associated with the fund, and finally the Fund row itself.
+
+run `rails c` again
+
+```ruby
+# delete the PaperTrails
+#  🚨 ⚠️ 🚨 ⚠️ 🚨 This is the point of no return.
+# Once these are deleted, we cannot restore the funds information from this database
+defunct_fund_versions = PaperTrail::Version.where_object(fund_id: id)
+defunct_fund_versions.destroy_all
+defunct_fund_changes = PaperTrailVersion.where_object_changes(fund_id: id)
+defunct_fund_changes.destroy_all
+
+# Finally, destroy the fund row
+defunct_fund_id = #the fund you're removing!
+fund = Fund.find(defunct_fund_id)
+fund.destroy
+
+exit
+```
+
+Copy any exerpts from the output files you want for later reference and `exit` the bash
+
+### Removing the Subdomain Configuration
+
+#### Remove Google OAuth
+  -  Go to our [GCP project](https://console.cloud.google.com/apis/credentials?project=daria-services-multitenant)
+  -  Open up DARIA Services Production from the list of Client IDs to edit
+  -  Remove the Authorized redirect URI for the fund's subdomain
+#### Remove the DNS in Namecheap
+  -  Open up the Namecheap control panel and head to Advanced DNS.
+  -  Remove the CNAME Record for the host 
+#### Remove the subdomain in heroku
+  -  Open up the pipeline in Heroku and go to the production instance, then open up Settings, then go to Domains.
+  -  Click the edit icon for the fund's domain and select Remove Domain
+  - Remove the domain's certs, if not done automatically
+#### Remove defunct manual database backups in heroku
+  -  Open up the pipeline in Heroku and go to the production instance, then open up Resources, then click on Heroku Postgres, then open Durability
+  - Remove any Manual Backups & Data Exports
```

Comment:
We have a few of these hanging out, all 6-9 months old. I think we have to have this step to ensure we won't retain copies. Are there any caveats we want to add for technical needs?

Reply:
nah I think this is a good call, though it does make me think 'maybe start this process by taking a db snapshot' is a good idea?